moveit_planners/pilz_industrial_motion_planner/src/trajectory_generator.cpp

```diff
@@ -326,7 +330,7 @@ bool TrajectoryGenerator::generate(const planning_scene::PlanningSceneConstPtr&
     return false;
   }
 
-  MotionPlanInfo plan_info;
+  MotionPlanInfo plan_info(scene, req);
   try
   {
     extractMotionPlanInfo(scene, req, plan_info);
```

Comment:
```suggestion
    extractMotionPlanInfo(plan_info.start_scene, req, plan_info);
```

Reply:
This fixes the `z=0` problems I'm seeing.

Reply:
Unsure if this change should also be made to `plan` on line 349?

Reply:
Yes, definitely!